encoding/kzg/prover/multiframe_proof.go

```diff
@@ -13,66 +14,18 @@ import (
 	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 )
 
-type KzgCpuProofDevice struct {
+type KzgMultiProofGnarkBackend struct {
```

Comment:
Shall we create a `gnark` folder similar to `icicle` to organize these implementations with gnark?